examples/python/using_carrays.py

```diff
@@ -24,8 +24,11 @@
 
```

Comment:
OK to remove, we have plenty of examples for this within the drivers that actually use it.